tool/tsh/proxy.go

```diff
@@ -564,25 +565,73 @@ Use the following command to connect to the database:
   $ {{.command}}
 `))
 
+const (
+	envVarFormatText                 = "text"
+	envVarFormatUnix                 = "unix"
+	envVarFormatWindowsCommandPrompt = "command-prompt"
+	envVarFormatWindowsPowershell    = "powershell"
+)
+
+var envVarFormats = []string{
+	envVarFormatText,
+	envVarFormatUnix,
+	envVarFormatWindowsCommandPrompt,
+	envVarFormatWindowsPowershell,
+}
+
+func envVarFormatFlagDescription() string {
+	return fmt.Sprintf(
+		"Optional format to print the commands for setting environment variables, one of: %s.",
+		strings.Join(envVarFormats, ", "),
+	)
+}
+
+// envVarCommand returns the command to set environment variables based on the
+// format.
+//
+// https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-envvars.html
+func envVarCommand(format, key, value string) (string, error) {
+	switch format {
+	case envVarFormatText, "":
+		return fmt.Sprintf("%s=%s", key, value), nil
+
+	case envVarFormatUnix:
+		return fmt.Sprintf("export %s=%s", key, value), nil
+
+	case envVarFormatWindowsCommandPrompt:
+		return fmt.Sprintf("set %s=%s", key, value), nil
```

Comment:
What do you think about handing missing `--format` flag by checking the ` runtime.GOOS` and returning by default platform specific export env formatting for `Unix` `Darwin` and `Windows` ? .